exercises/luhn/cases_test.go

```diff
@@ -39,6 +39,11 @@ var testCases = []struct {
 		"055 444 286",
 		false,
 	},
+	{
+		"valid number with an even number of digits",
+		"095 245 88",
+		true,
+	},
```

Comment:
Thanks for your feedback.  This will be a good additional test case.  As in #1180, it's a generated file.